sdk/core/core-lro/review/core-lro.api.md

```diff
@@ -10,11 +10,26 @@ import { AbortSignalLike } from '@azure/abort-controller';
 export type CancelOnProgress = () => void;
 
 // @public
-export interface LongRunningOperation<T> {
-    requestMethod: string;
-    requestPath: string;
-    sendInitialRequest: () => Promise<LroResponse<T>>;
-    sendPollRequest: (path: string) => Promise<LroResponse<T>>;
+export function createHttpPoller<TResult, TState extends OperationState<TResult>>(lro: LongRunningOperation, options?: CreateHttpPollerOptions<TResult, TState>): Promise<SimplePollerLike<TState, TResult>>;
+
+// @public
+export interface CreateHttpPollerOptions<TResult, TState> {
+    intervalInMs?: number;
+    processResult?: (result: unknown, state: TState) => TResult;
+    resourceLocationConfig?: LroResourceLocationConfig;
+    restoreFrom?: string;
+    updateState?: (state: TState, response: LroResponse) => void;
+    withOperationLocation?: (operationLocation: string) => void;
+}
+
+// @public
+export interface LongRunningOperation<T = unknown> {
+    requestMethod?: string;
```

Comment:
make sure it is case-insensitive

Reply:
done in https://github.com/Azure/azure-sdk-for-js/pull/22753/commits/9e14a3854639052183b19b6595579acdc7a2404f